src/main/java/net/dries007/tfc/common/blockentities/LoomBlockEntity.java

```diff
@@ -57,7 +57,6 @@ public static void tick(Level level, BlockPos pos, BlockState state, LoomBlockEn
                         loom.inventory.setStackInSlot(SLOT_RECIPE, ItemStack.EMPTY);
                         loom.inventory.setStackInSlot(SLOT_OUTPUT, recipe.assemble(new ItemStackInventory(loom.inventory.getStackInSlot(SLOT_RECIPE))));
                     }
-                    loom.markForSync();
```

Comment:
How does *removing* a call to sync fix a sync issue? I'm also very suspicious that this is the right fix given you explicitly haven't tested it on dedicated server.